### PR TITLE
Refactor/musician card and profile

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- n/a

What (if anything) did you refactor?
- renamed the component being used in the SubmissionDetails component from MusicianCard to MusicianProfile. I realized the MusicianCard was meant to go with the AdminPage and I forgot to use that component, and the MusicianProfile was meant to go with the submission details page. I didn't change any of Emma's code that was in MusicianCard, just updated that component folder, files, and imports to be MusicianProfile instead.
- moved the logic for rending the cards on the admin page from AdminPage to MusicianCard, I realized this was the plan we had talked about and I just forget to utilize the MusicianCard when I was building out AdminPage
- updated sample data to make all ids unique, it was throwing an error since not all keys on the musician cards were unique
- refactored the scss for SubmissionForm: the h2 rule was effecting h2s in other elements. With the way scss compiles all of the files together, we will need to either A) nest any element that doesn't have a classname inside it's parent class (ex: nest an h2 rule in the "admin-container" rule, this would only add that styling rule to h2s inside the admin-container rather than being applied to all h2s across the app), or B) give all elements a unique classname and use that classname in the scss file. This ensures that you are only targeting that specific element rather than all elements of that type.
- updated the paths in each route to be lowercase, I think that seems like best practice 

Were there any issues that arose?
- none: I opened the app and made sure everything still looked the same, looks good! 

Is there anything that you need from your teammate?
- n/a 

Any other comments, questions, or concerns?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ function App() {
       <main>
         <Routes>
           <Route exact path='/' element={<LandingPage />} />
-          <Route path='/Admin' element={<AdminPage />} />
-          <Route path='/Submission' element={<SubmissionForm />} />
-          <Route path='/SubmissionDetails' element={<SubmissionDetailsPage />}
+          <Route path='/admin' element={<AdminPage />} />
+          <Route path='/submission' element={<SubmissionForm />} />
+          <Route path='/submissiondetails' element={<SubmissionDetailsPage />}
```
Comment (mr-ryan12) [inline src/App.js:20]: I agree, this seems like best practice. Nice catch.

File: src/components/SubmissionForm/SubmissionForm.scss [hunk 2]
```diff
@@ -17,20 +17,22 @@ form {
...
 
-h2 {
-  font-size: 3vh;
-  color: #FFF;
-  text-align: center;
+  h2 {
+    font-size: 3vh;
+    color: #FFF;
+    text-align: center;
+  }
```
Comment (mr-ryan12) [inline src/components/SubmissionForm/SubmissionForm.scss:31]: Neat! Thank you for making this switch and teaching me a new CSS trick! I definitely should have targeted the element by className. Thank you for catching that.

**Review by mr-ryan12 — APPROVED** [review]
Well done. Many thanks for making these changes!

**Review by Echory — APPROVED** [review]
Thanks for fixing those components ! Looks good.

**Review by oacortez — APPROVED** [review]
Thank you for creating this refactor branch, Ali! I agree with the changes that have been made. Thank you for fixing the route path to lower case and as well the other minor details. Everything looks good and I will merge this.

**Pull request merged**